dedibox-network/rpn/how-to/use-rpn-san-proxmox.mdx

```diff
@@ -79,7 +79,7 @@ Proxmox requires the Open-iSCSI software to support mounting of iSCSI volumes. I
 
 ## Discover and connect to targets in Proxmox
 
-1. In the **Server View**, click **Datacenter**, then **Storage**.
+1. In the **Server View**, click **Data center**, then **Storage**.
```

Comment:
If this is referring to the way it is written in another interface, we should maybe leave it the way it is?

Reply:
Well spotted. I'll fix it. Thanks, Ro!